cccatalog-api/cccatalog/api/controllers/search_controller.py

```diff
@@ -6,7 +6,7 @@
 from elasticsearch_dsl.query import Query
 from cccatalog import settings
 from django.core.cache import cache
-from cccatalog.api.models import ContentProvider
+import cccatalog.api.models as models
```

Comment:
Not sure if it's ok to revert to the previous import syntax to be like the others

Reply:
This is a specific choice to avoid circular import problems in Python (search_controller imports models and models imports search_controller). More details [here](https://stackoverflow.com/questions/22187279/python-circular-importing)